explorer/deploy/wizard.ts

```diff
@@ -0,0 +1,210 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Is this and `webAppCreator.ts` largely a fork of the App Service extension?

Reply:
yes, i'm hoping that we can consolidate all of these "creation" steps into a single package everyone can consume, but for now it is effectively a fork.

Reply:
Sounds good. I was just curious, so thanks for confirming. This change looks awesome!